include/seqan3/argument_parser/validators.hpp

```diff
@@ -264,15 +264,25 @@ class value_list_validator
  */
 //!\brief Deduction guide for a parameter pack over an arithmetic type.
 template <arithmetic ...option_types>
+//!\cond
+    requires !(std::same_as<char, option_types> || ...)
```

Comment:
it should 
```suggestion
    requires !(seqan3::detail::is_char_adaptation_v<option_types> || ...)
```

Actually didn't we have a char concept?

Reply:
I cannot find a concept. Just the templetized variable